.github/workflows/ci.yml

```diff
@@ -42,7 +42,11 @@ concurrency:
   # Cancel all workflow runs except latest within a concurrency group. This is achieved by defining a concurrency group for the PR.
   # Non-PR builds have singleton concurrency groups.
   # When triggered by the repository_dispatch, add the expected SHA to avoid cancelling the run from the PR.
-  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.event.number || github.sha }}-${{ github.event_name == 'repository_dispatch' && github.event.client_payload.slash_command.args.named.sha || github.sha }}
+  group: |
+    ${{ github.workflow }}-
+    ${{ github.event_name == 'pull_request' && github.event.number || '' }}-
+    ${{ github.event_name == 'repository_dispatch' && github.event.client_payload.slash_command.args.named.sha || '' }}-
+    ${{ github.event_name != 'pull_request' && github.event_name != 'repository_dispatch' && github.sha || '' }}
```

Comment:
Would it be useful to make this more "structured" to make it easier when debugging? The group name is logged when concurrency group takes some action.

Something like:

```suggestion
    workflow=${{ github.workflow }},
    pr_number=${{ github.event_name == 'pull_request' && github.event.number || 'NA' }},
    dispatch_sha=${{ github.event_name == 'repository_dispatch' && github.event.client_payload.slash_command.args.named.sha || 'NA' }},
    commit_sha=${{ github.event_name != 'pull_request' && github.event_name != 'repository_dispatch' && github.sha || 'NA' }}
```

Reply:
So much better, thanks!
![image](https://user-images.githubusercontent.com/795177/208402919-d7f2b918-6c01-4a46-bdd3-6a355cbe7ee2.png)